invokeai/frontend/web/src/features/gallery/components/ImageMetadataViewer/DataViewer.tsx

```diff
@@ -61,8 +61,11 @@ const DataViewer = (props: Props) => {
       >
         <OverlayScrollbarsComponent
           defer
-          style={{ height: '100%', width: '100%' }}
+          style={{ width: '100%' }}
```

Comment:
Removing `height: "100%"` here prevents the scrollbars from appearing - do you recall why this needed to be removed?

Reply:
while removing this parameter, metadata in its .json view becomes scrollable (leaving it disables scrolling when there is almost no place for showing it)

Reply:
Ok, I think we should solve that by making the json view area have a minimum height instead of changing the height attr.